client/blocks/plugins-scheduled-updates-multisite/index.tsx

```diff
@@ -47,7 +48,11 @@ export const PluginsScheduledUpdatesMultisite = ( {
 
 	return (
 		<MultisitePluginUpdateManagerContextProvider>
-			<Layout title={ title } wide>
+			<Layout
+				className={ clsx( 'sites-dashboard', 'sites-dashboard__layout' ) }
```

Comment:
Why 'sites-dashboard' and 'sites-dashboard__layout' here? This doesn't seem to follow Calypso's CSS guidelines.

Reply:
I just followed another plugin page [here](https://github.com/Automattic/wp-calypso/blob/88e761dc00a46a92b1d60dcc9280d7670a818df9/client/my-sites/plugins/plugins-dashboard/index.tsx#L328). But yes, it's not an ideal solution... Let me think of a better approach 🤔

Reply:
What is the "Layout" component, should that component apply the right styles without extra classnames? I'm not very familiar with this part of the code to provide the right answer here. Maybe it's ok for now, but something is off.

Reply:
I reverted this change by https://github.com/Automattic/wp-calypso/pull/103232/commits/84734879686a09ab15d8605625ada3fff3ae1de9 after realizing it was no longer necessary due to recent updates I made to the styles